deposit-model/src/main/java/org/dataconservancy/nihms/model/DepositMetadata.java

```diff
@@ -274,16 +301,42 @@ public void setDoi(URI doi) {
         public String email;
 
         /**
-         * Principle Investigator (i.e. NIHMS "funding provider")
+         * Role for this entry for this person (multiple entries allowed per person)
```

Comment:
@Trumbore what does multiple entries mean?  That a single instance of `Person` contains multiple values for `type`? Or that there are multiple instances of a `Person` (all representing the same human individual), each instance with a different type?

Reply:
This is important because someone using this metadata needs to know if they should expect a person to show up multiple times in a list, for example.

The more intuitive behavior (IMHO) would be to have a single Person instance that carries multiple values for type.

Reply:
I will update the javadoc to make this more clear.  There could be entries in the People list that represent the same human being in different roles.  Because the person data comes from three different places (Submission, Grant, Submission metadata) and is formatted alternately as first/middle/last names and as a single name string, it would be very hard to reliably establish correlations between these people once we get to the deposit service.

Having the metadata use a single name string is a particular problem as it invites the person who enters it to deviate from what they might enter if asked for all three names.  Any extra or missing spaces, periods, middle names, etc. would make it hard to see that two people are the same.  If we could identify matching human beings, then yes, storing a list of roles with each Person would be an ideal solution.